example/dj/urls.py

```diff
@@ -11,5 +13,7 @@
     url(r'^api/extra/$', ExtraResource.as_view())
 ]
 
-if django.get_version() < '1.9':
+if StrictVersion(django.get_version()) < StrictVersion('1.9'):
```

Comment:
1. to sice ano ale jen pro určité případy, tady prostě potřebuji pro každou verzi djanga konkrétní chování, není to situace kdy kontroluji zda existuje nainstalovaná knihovna, to vím jen potřebuji měnit chování na základě konkrétní verze. Což souvisí i s 2, radil jsem ti to pro ověření zda je security nainstalovaná to bys jinak neudělal
4. bohužel tady to musím fixovat na verzi to je přesně to co je cílem souboru compatibility :)